### DISCUSSION TITLE
Reading data from multiple spaces

### DISCUSSION THREAD

**Discussion by ShivaniSK** [finos/TimeBase-CE#222] [General]
Hello,
Currently we are reading from Space like below

```
SelectionOptions opts = new SelectionOptions();
opts.space = "2020"

```
Is there a possibility to read data from multiple Spaces by comma separating the space names like below?
`opts.space = "2020,2021"`

**Comment by alex-karpovich** [comment]
Greetings @ShivaniSK - yes it possible. next week I will try to add

**Reply by ShivaniSK** [reply]
Thanks @alex-karpovich . Please do let me know once it's added.
Appreciate your support.

**Comment by ShivaniSK** [comment]
Hi @alex-karpovich , do we have this feature available? Can you please confirm?

**Reply by alex-karpovich** [reply]
yes, i will publish new version on 31 May

**Reply by ShivaniSK** [reply]
great thanks alot @alex-karpovich . Please let us know once its published.

**Reply by ShivaniSK** [reply]
Hi @alex-karpovich , do we have the new version published?

**Reply by alex-karpovich** [reply]
Greetings, @ShivaniSK please use 6.1.13

https://github.com/finos/TimeBase-CE/releases/tag/6.1.13

**Reply by ShivaniSK** [reply]
Thanks @alex-karpovich , do we have any release notes or document which talks about how to use multiple spaces option?

**Reply by alex-karpovich** [reply]
Please use ```SelectionOptions.spaces``` or ```SelectionOptions.withSpaces()```
to define your subscription.

Check here:
https://github.com/finos/TimeBase-CE/blob/ad5ef9a34d436a60b83404e6c0fded20552c6323/java/timebase/api/src/main/java/com/epam/deltix/qsrv/hf/tickdb/pub/SelectionOptions.java#L105

**Reply by ShivaniSK** [reply]
Thanks alot :)
